Month1/Week2/Section1/Exercise3/Main.cpp

```diff
@@ -0,0 +1,154 @@
+/*
+Exercise 3 - Shapes and Areas
+
+Create two classes: a Circle class and a Square class. For each class, take in the necessary information to calculate its area and perimeter.  Remember to validate input.
+
+Ask the user if they want to create a circle or a square. For their pick, ask the necessary information to create the shape. Then, tell them the area and the perimeter of their shape.
+
+Your class should have the following methods:
+ExampleShape();
+ExampleShape( Params );
+float GetArea();
+float GetPerimeter();
+
+Example output
+Welcome to create - a - shape, what type of shape do you want ?
+1 - Circle
+2 - Square
+Your choice : 1
+
+Enter radius for your circle : 30
+Your shape�s area : 2827.43
+Your shape�s perimeter : 188.5
+
+
+*/
+
+
+#include <iostream>
+#include <cmath>
+#include <iomanip>
+
+
+class Circle
+{
+private:
+	
+	float radius;
+
+public:
+
+	Circle()
+	{
+		radius = 0.0f;
+	}
+
+	Circle(float pRadius)
+	{
+		radius = pRadius;
+	}
+
+	float getArea() const
+	{
+		return 3.14159265f * std::pow(radius, 2);
+	}
+
+	float getPerimeter() const
+	{
+		return 2 * 3.14159265f * radius;
+	}
+
+};
+
+
+class Square
+{
+private:
+
+	float sideLength;
+
+public:
+
+	Square()
+	{
+		sideLength = 0.0f;
+	}
+
+	Square(float pSideLength)
+	{
+		sideLength = pSideLength;
+	}
+
+	float getArea() const
+	{
+		return std::pow(sideLength, 2);
+	}
+
+	float getPerimeter() const
+	{
+		return 4 * sideLength;
```

Comment:
Remember we use .f for floats. If you want to output a float, why use an integer instead of a float for this multiplication? Same as the rest of integers in this file that could be floats.